packages/babel-plugin/src/css-prop/__tests__/object-literal.test.ts

```diff
@@ -125,7 +125,7 @@ describe('css prop object literal', () => {
     `);
 
     expect(actual).toInclude(
-      'background-image:linear-gradient(45deg,gray 25%,transparent 25%),linear-gradient(-45deg,gray 25%,transparent 25%),linear-gradient(45deg,transparent 75%,gray 75%),linear-gradient(-45deg,transparent 75%,gray 75%)'
+      'background-image:linear-gradient(45deg,gray 25%,transparent 0),linear-gradient(-45deg,gray 25%,transparent 0),linear-gradient(45deg,transparent 75%,gray 0),linear-gradient(-45deg,transparent 75%,gray 0)'
```

Comment:
These values are indeed the same 
![Screen Shot 2022-08-05 at 4 33 02 pm](https://user-images.githubusercontent.com/14813563/183016238-755188a7-dfa0-4f35-93c1-b0739831babf.png)